src/sdk/services/blockchain/contracts/VestingContract.ts

```diff
@@ -162,4 +163,14 @@ export class VestingContract {
 
         return res.rows;
     }
+
+    async getBalance(account: NameType): Promise<number> {
+        const assets = await (
+            await getApi()
+        ).v1.chain.get_currency_balance(CONTRACT_NAME, account, getSettings().currencySymbol);
```

Comment:
no this will not work. The user's balance is inside the allocation table, not associated to their name. You will need to use the `getAllocations()` function, then iterate through all the allocations to sum up the total LEOS.

Reply:
add a unit test to the suite to test this out and know it is working.